### ISSUE TITLE
Register User

### ISSUE THREAD

**Issue by tabaktoni** [MacKentoch/react-redux-graphql-apollo-bootstrap-webpack-starter#6]
I have this error when i try to test register user from app:
GraphQL error: User is not permitted to access mutation: createUser

**Comment by MacKentoch** [comment]
Hi,

Did you set configuration after [creating an account](https://github.com/MacKentoch/react-redux-graphql-apollo-bootstrap-webpack-starter#server)?

**Comment by tabaktoni** [comment]
Yes, i also tested directly on scaphold graphiQl and it worker and login worked. But when i try from test app i get that error. 

This is what i did:
1. npm install
2. Setup app on scaphold "as is"
3. change config/index.js with API endpoint
```
export const appConfig = {
  // apollo client:
  apollo: {
    networkInterface: 'https://us-west-2.api.scaphold.io/graphql/my-booker'
  }
};
```
4. npm run start

**Comment by MacKentoch** [comment]
Does it work with the [preview](https://mackentoch.github.io/react-redux-graphql-apollo-bootstrap-webpack-starter/#/)?

I don't see any particular reason why it would not work.

**Comment by abh4chat** [comment]
i bypassed this issue by toggling permissions

**Comment by MacKentoch** [comment]
Which permissions are you referring to?

**Comment by abh4chat** [comment]
Permission scope on type in scaphold.

**Comment by MacKentoch** [comment]
That makes sense.

Thank you @abh4chat for sharing your feedback.
